pkg/pb/pointerdb.proto

```diff
@@ -101,6 +101,16 @@ message ListRequest {
   bytes API_key = 7;
 }
 
+//IterateRequest is a request message for the iterate private method
+message IterateRequest {
```

Comment:
Iteration isn't the purpose for the message. It's more about requesting things that need repair.

Reply:
actually, the purpose of this message is to iterate over all objects. we don't know what things need repair yet. the checker needs to iterate over everything

Reply:
I was thinking that it will internally iterate and then return the items that need repairing. At least that's what the implementation is doing.

Reply:
oh nevermind! you're right. i was working off the assumption that this pr was just implementing the one ticket, but it looks like this also does the checking of what needs repair too!

Reply:
yeah it seemed easy enough to combine the tickets, hope thats ok